gulpfile.js

```diff
@@ -7,36 +7,36 @@
    To add a new task, simply add a new task file the gulp/tasks directory.
```

Comment:
Like we discussed on our call, I wouldn't mess with changing this back right now, but the formatting changes that your editor made to this file and the one above it are going to potentially confuse reviewers at CFPB, who may think you changed stuff that you really didn't.

Reply:
You may need to run it through Prettier with different settings at some point to fix the formatting

Reply:
OK